src/services/OswServiceTaskConsole.cpp

```diff
@@ -126,6 +126,9 @@ void OswServiceTaskConsole::runPrompt() {
             NotifierClient notify("osw.console");
             notify.showToast(toast);
 #endif
+        } else if (this->m_inputBuffer.find("notify ") == 0 and this->m_inputBuffer.length() > 7) {
+            auto value = this->m_inputBuffer.substr(7);
+            OswUI::getInstance()->showNotification(value, false);
```

Comment:
we already have a `toast` command, what does this contribute then?

Reply:
Just in case there are multiple situations in the future, I expect there to be a test mode in various cases. What do you think?

Reply:
In the end, they just show both a toast-notification. I think the bug #414 should be fixed instead of providing **two** commands which do exactly the same thing (result-wise). Even for testing, I would recommend more a unit-test instead of adding this command.